### ISSUE TITLE
Asset linking issue with sprockets-rails

### ISSUE THREAD

**Issue by tarebyte** [github/octicons#50]
I'm currently using octicons with bower in the Rails asset pipeline with `sprockets-octicons`, I updated sprocket-rails to `2.3.0` and now I have the following issue

```
Asset was linked to from an alias rather than its exact path. Alias resolving may not be available in production.
Use "octicons/octicons/octicons.eot" instead of "octicons.eot"
```

I changed `sprockets-octicons.scss` to
```scss
@font-face {
  font-family: 'octicons';
  src: font-url('octicons/octicons/octicons.eot?#iefix') format('embedded-opentype'),
         font-url('octicons/octicons/octicons.woff') format('woff'),
         font-url('octicons/octicons/octicons.ttf') format('truetype'),
         font-url('octicons/octicons/octicons.svg#octicons') format('svg');
  font-weight: normal;
  font-style: normal;
}
```

And it works again, but I don't know if that's a good final solution. I'd be happy to make a PR to help out though!

**Comment by cameronmcefee** [comment]
@tarebyte are you using octicons via sprockets using the instructions provided in the readme, or through some alternative method?

**Comment by tarebyte** [comment]
I'm using octicons through the instructions on the README for bower with sprockets. My octicons folder is under `vendor/assets/bower_components/octicons`

These are my configurations:
```ruby
# config/application.rb

require File.expand_path('../boot', __FILE__)

require 'rails/all'

Bundler.require(*Rails.groups)

module Classroom
  class Application < Rails::Application
    # Do not swallow errors in after_commit/after_rollback callbacks.
    config.active_record.raise_in_transactional_callbacks = true

    # Add bower assets to the path
    config.assets.paths << Rails.root.join('vendor', 'assets', 'bower_components')

    # Precompile Fonts
    config.assets.precompile += %w(*.svg *.eot *.woff *.ttf)
  end
end
```

```scss
// application.css.scss

/*
 *= require octicons/octicons/sprockets-octicons
 *= require_self
 *= require_tree .
 */

@import "primer-css/scss/primer";
```

**Comment by cameronmcefee** [comment]
@josh would you be able to provide some clarity here? Is this an issue that needs to be resolved in this repo, or is this an implementation-specific issue?

**Comment by josh** [comment]
Ugh, this is a sprockets-rails bug (my fault actually). That check should only be applying to `image_path` helpers in ActionView, not inside Sass `font-url` itself.

**Comment by josh** [comment]
Opened up https://github.com/rails/sprockets-rails/issues/247.

**Comment by josh** [comment]
This should be fixed in sprockets-rails 2.3.1.

**Comment by tarebyte** [comment]
Updated to sprockets-rails 2.3.1, it now works for me. Thanks @josh!
